expandybird/expansion/expansion.py

```diff
@@ -183,6 +194,99 @@ def _ValidateUniqueNames(template_resources, template_name='config'):
     # If this resource doesn't have a name, we will report that error later
 
 
+def IsTemplate(resource_type):
```

Comment:
That doesn't make sense. You can import text files and python libraries that aren't templates.